OracleWebLogic/samples/1221-multihost/create-container.sh

```diff
@@ -23,6 +24,8 @@ fi
 eval "$(docker-machine env $swarm $machine)"
 
 docker run -d $DOCKER_CONTAINER_INSTANCE_OPTIONS \
+  -e MS_HOST=$name\
```

Comment:
Yes I had to set the Managed Server host name so that I could access the servers from the HttpServletProxy application using a host name.  When we run the managed servers in a Docker Container we do not configure the listen address, thus we need to use a hostname to reach the servers.